app/src/main/java/com/android/solvit/MainActivity.kt

```diff
@@ -9,35 +9,37 @@ import androidx.compose.material3.Surface
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.semantics.semantics
-import androidx.compose.ui.semantics.testTag
-import androidx.compose.ui.tooling.preview.Preview
-import com.android.solvit.resources.C
 import com.android.solvit.ui.theme.SampleAppTheme
 
 class MainActivity : ComponentActivity() {
+
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
     setContent {
       SampleAppTheme {
-        // A surface container using the 'background' color from the theme
-        Surface(
-            modifier = Modifier.fillMaxSize().semantics { testTag = C.Tag.main_screen_container },
-            color = MaterialTheme.colorScheme.background) {
-              Greeting("Android")
-            }
+        Surface(modifier = Modifier.fillMaxSize(), color = MaterialTheme.colorScheme.background) {
+          Greeting()
+        }
       }
     }
   }
 }
 
 @Composable
-fun Greeting(name: String, modifier: Modifier = Modifier) {
-  Text(text = "Hello $name!", modifier = modifier.semantics { testTag = C.Tag.greeting })
+fun Greeting() {
+  Text(text = "Hello, World!")
 }
-
-@Preview(showBackground = true)
+/*
 @Composable
-fun GreetingPreview() {
-  SampleAppTheme { Greeting("Android") }
-}
+fun Greeting() {
```

Comment:
The name of this function should be changed to something more clear like "SolvitApp"